Azure/azure-cli
# [Security] az security pricing: Change help to reflect current API version being called

**Description**  
Update az security pricing help with correct documentation

**Testing Guide**  
N/A

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
